client/src/components/exercise/lab6/pages/Employer/AIRepair.js

```diff
@@ -0,0 +1,326 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable require-jsdoc */
+import { navigate } from "@reach/router";
+import React, { useEffect, useState } from "react";
+import { EXERCISE_PLAYING } from "../../../../../constants/lab6";
+import RepairService from "../../../../../services/lab6/RepairService";
+import CodeUpdateHeader from "../../../lab3/components/CodeUpdateHeader";
+import Popup from "../../../shared/Popup";
+
+const AIRepair = (props) => {
+  const { actions, user } = props;
+
+  useEffect(() => {
+    actions.updateState(EXERCISE_PLAYING);
+  }, [actions]);
+
+  const [repairOpen, setRepairOpen] = useState(false);
```

Comment:
Should we use a redux reducer here for all the states?

Reply:
> Should we use a redux reducer here for all the states?

@notcruz I mean it could be beneficial I am not entirely sure how redux works so I steered away from using it in this lab  to avoid craziness

Reply:
Redux adds all the states and their dispatches (update functions) to a single file. We can look into updating this in the future, it's just a matter of reducing complexity.